src/actions/profile-view.js

```diff
@@ -137,9 +137,11 @@ export function selectLeafCallNode(
     const filteredThread = threadSelectors.getFilteredThread(getState());
     const callNodeInfo = threadSelectors.getCallNodeInfo(getState());
 
+    // The newSelectedStack could be undefined if there are 0 samples.
     const newSelectedStack = filteredThread.samples.stack[sampleIndex];
+
     const newSelectedCallNode =
-      newSelectedStack === null
+      newSelectedStack === null || newSelectedStack === undefined
```

Comment:
thanks for the fix !!
curious that it wasn't deterministic and only happened sometimes...

Reply:
However, that was a lot more work with more call sites, this fix was expedient.